cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/AbstractSchedulerService.java

```diff
@@ -169,6 +171,7 @@ public void resumeSchedule(Id.Program program, SchedulableProgramType programTyp
       scheduler.resumeSchedule(program, programType, scheduleName);
     } catch (NotFoundException e) {
       LOG.trace("Could not resume schedule", e);
+      Throwables.propagate(e);
```

Comment:
Why propagate this as a runtime exception? What is wrong with re-throwing the NotFoundException?